pkg/config/config.go

```diff
@@ -16,20 +16,21 @@ var Cfg = new()
 
 // Define a Config type to hold our config properties.
 type Config struct {
-	API_SERVER_URL string // address for Kubernetes API Server
-	AuthCacheTTL   int    // Time-to-live (milliseconds) of Authentication (TokenReview) cache.
-	SharedCacheTTL int    // Time-to-live (milliseconds) of common resources (shared across users) cache.
-	UserCacheTTL   int    // Time-to-live (milliseconds) of namespaced resources (specifc to users) cache.
-	ContextPath    string
-	DBHost         string
-	DBName         string
-	DBPass         string
-	DBPort         int
-	DBUser         string
-	HttpPort       int
-	PlaygroundMode bool // Enable the GraphQL Playground client.
-	QueryLimit     int  // The default LIMIT to use on queries. Client can override.
-	RelationLevel  int  // The number of levels/hops for finding relationships for a particular resource
+	API_SERVER_URL    string // address for Kubernetes API Server
+	AuthCacheTTL      int    // Time-to-live (milliseconds) of Authentication (TokenReview) cache.
+	SharedCacheTTL    int    // Time-to-live (milliseconds) of common resources (shared across users) cache.
+	UserCacheTTL      int    // Time-to-live (milliseconds) of namespaced resources (specifc to users) cache.
+	PropTypesCacheTTL int    // Time-to-live (milliseconds) of property datatypes cache in shared cache.
```

Comment:
I'd reuse `SharedCacheTTL` to keep it simple.
This is better, but I'm afraid that too many different TTL will create confusion.

Reply:
sounds good.